Sources/DistributedSystem/DistributedSystem.swift

```diff
@@ -181,9 +187,16 @@ public class DistributedSystem: DistributedActorSystem, @unchecked Sendable {
         case serviceForRemoteClient(Channel, AsyncStream<InvocationEnvelope>.Continuation, ManagedAtomic<UInt64>)
     }
 
+    struct ChannelInfo {
+        var bytesReceived = 0
```

Comment:
Maybe some description of these fields?

Reply:
Well... is it really not clear from the name?